Limitations.md

```diff
@@ -255,10 +255,10 @@ This section lists limitations where the possible solutions are uncertain.
 ## Docker --security-opt option partially supported
 
 The `--security-opt=` option used by Docker is partially supported.
-We only support --security-opt=no-new-privileges option as of today.
-
-See https://github.com/kata-containers/kata-containers/issues/31 for more information.
+We only support --security-opt=no-new-privileges and security-opt seccomp=/path/to/seccomp/profile.json
+option as of today.
 
+Note: The `--security-opt apparmor=your_profile` is yet not supported. See https://github.com/kata-containers/runtime/issues/707
```

Comment:
- I think "not yet" rather than "yet not" ready more clearly.
- Nit: Missing period at end of sentence.